sound150@claudiux/files/sound150@claudiux/metadata.json

```diff
@@ -14,5 +14,5 @@
         "3.6",
         "3.8"
     ],
-    "last-edited": 1523640550
+    "last-edited": 1526065577
```

Comment:
Maybe try removing this from your metadata.json? Not sure why the site won't update, but Cinnamon is supposed to handle the `last-edited` field internally.